src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServicesCmdletBase.cs

```diff
@@ -138,6 +140,7 @@ protected override void StopProcessing()
             }
             catch (Exception ex)
             {
+                this.WriteVersionInfoToDebugChannel();
```

Comment:
Yep, that will be a bigger change: derive from ErrorRecord, put the module version in it and return that object. Let me do that when we have more information needs to be displayed during exception handling.